src/app/components/MediaPlayer/AudioLoader/index.jsx

```diff
@@ -49,6 +54,7 @@ const AudioLoader = ({ children, isLoading }) => (
           height="38"
           viewBox="0 0 38 38"
           focusable="false"
+          stroke="black"
```

Comment:
Is this for no CSS?

Reply:
Guess that means it will always have a black stroke? Or will the stroke above with current colour take presidence when you have CSS?

Reply:
Yep :)

Reply:
> Guess that means it will always have a black stroke? Or will the stroke above with current colour take presidence when you have CSS? Just wondering what it will look like in Firefox when the user changes their colour preferences...

CSS overrides the attribute, so with CSS enabled the spinner will look the same as it does now